templates/endpoints_v2/examples/OptimizeModels.ipynb

```diff
@@ -19,10 +19,10 @@
     "\n",
     "## Configurations to optimize\n",
     "\n",
-    "These are some configurations you should consider changing when updating tensor parallelism or the accelerator type:\n",
-    "1. The `num_workers` configuration can be used to set the tensor parallelism for a model. A higher value for tensor parallelism will typically lead to lower latency at the cost of more GPUs per replica.\n",
+    "These are some configurations that are updated for different tensor parallelism or hardware type:\n",
+    "1. The `num_workers` configuration is used to set the tensor parallelism for a model. A higher value for tensor parallelism will typically lead to lower latency at the cost of more GPUs per replica.\n",
     "2. `resources_per_worker` in `scaling_config` and `resources` under `ray_actor_options` in the `deployment_config`. These determine the accelerator type used. It must follow the format of `\"accelerator_type:T4\":0.01`.\n",
-    "3. `engine_kwargs`: `max_num_batched_tokens` and `max_num_seqs`. These are the maximum number of batched tokens and sequences configured for each iteration in [vLLM](https://docs.vllm.ai/en/latest/models/engine_args.html). With increase in available GPU memory, you can increase these values.\n",
+    "3. `engine_kwargs`: The full list of available engine arguments when using vLLM are [here](https://docs.vllm.ai/en/latest/models/engine_args.html). There are various configurations to consider while optimizing. For example, `max_num_batched_tokens` and `max_num_seqs`. These are the maximum number of batched tokens and sequences configured for each iteration in vLLM. With increase in available GPU memory, you can increase these values. \n",
```

Comment:
Should we use the stable URL instead of the latest (i.e. `https://docs.vllm.ai/en/stable/models/engine_args.html`)?